ens/ens.py

```diff
@@ -345,10 +345,9 @@ def get_text(self, name: str, key: str) -> str:
             the "0x59d1d43c" interface id
         :raises ResolverNotFound: If no resolver is found for the provided name
         """
-        node = raw_name_to_hash(name)
-        normal_name = normalize_name(name)
```

Comment:
`resolver()` normalizes the name already. So we're normalizing a name to normalize it again. I think we could add some tests to make sure every method that is expected to normalize a name does indeed normalize it just in case it gets taken out at some point then the tests would fail. I'll create an issue for that.

Reply:
Ah, gotcha. In `async_ens.py` I believe we're doing a `normalize_name` still. Do you mind standardizing that in `async_ens`?

Reply:
Good call, can do 👍🏼